### DISCUSSION TITLE
Re-Onboarding is not triggered

### DISCUSSION THREAD

**Discussion by theurichde** [renovatebot/renovate#22982] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

Placeholder value, please select the correct response from the dropdown

### Describe the problem

Hi there!

Somehow Renovate didn't onboard our repository properly and nuking + re-onboarding does not work.

I already removed, waited and added the repo to the Renovate App, nothing changed so far.

After merging the initial onboarding PR, Renovate didn't run on a regular basis, that's why we wanted to re-onboard it.

onboarding job id: 0188e345-a0dd-7f58-8ebd-dc9af92ae7ab
second run job id: 8eabeb0c-59fc-43a3-80a8-f5bab063f7c1

### Relevant debug logs

<details><summary>Logs</summary>

From the last run (8eabeb0c-59fc-43a3-80a8-f5bab063f7c1) before nuking the config / renaming the onboarding PR

```
DEBUG: config file not found
DEBUG: Found closed onboarding PR
DEBUG: Repo is not onboarded and no merged PRs exist
DEBUG: Getting comments for #5
DEBUG: Found 1 comments
DEBUG: Ensuring comment "Renovate is disabled" in #5
INFO: Comment added
INFO: Repository is disabled - skipping
DEBUG: Repository result: disabled-closed-onboarding, status: disabled, enabled: false, onboarded: undefined
DEBUG: Repository timing splits (milliseconds)
DEBUG: Package cache statistics
DEBUG: http statistics
DEBUG: Package lookup durations
DEBUG: dns cache
INFO: Repository finished
DEBUG: Renovate exiting
```

</details>

### Have you created a minimal reproduction repository?

I have explained in the description why a minimal reproduction is impossible

**Comment by nabeelsaabna** [comment]
hi @theurichde 
After receiving this message `disabled-closed-onboarding` the repo will be marked as `disabled` disable as you would expect.
The team didn't manage to reproduce the problem "_nuking + re-onboarding does not work._" when flowing the described steps.
Did you merge/push a `renovate.json` file in the base branch after the second job was complete ?

**Reply by theurichde** [reply]
Hey there. 
The `renovate.json` was deleted the moment the 2nd run was triggered (I guess the deletion triggered the run?)

We did not push a new `renovate.json`after the 2nd run. Shall we try that?

**Reply by nabeelsaabna** [reply, marked as answer]
> I guess the deletion triggered the run?

Yes, the second the run that disabled the repo

> We did not push a new `renovate.json` after the 2nd run. Shall we try that?

Yes

**Reply by theurichde** [reply]
Re-adding `renovate.json` triggered the run, repo is onboarded and PRs are created. 

Thank you ❤️!

**Reply by nabeelsaabna** [reply]
Enjoy renovate 😄
